Extras/FrontendTests/playwright.config.ts

```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : 5,
+  /* Opt out of parallel tests in general as multiple streamers mean they can connect to the wrong test */
+  workers: 1,
```

Comment:
This was making tests flaky in CI

Reply:
I don't think so. The way the tests work they rely on an external SS and JSStreamer to be run. This means if two tests ever run in parallel they will both connect to these external services. This was messing up the tests in some cases. Without making it so each tests starts its own SS and JSStreamer the easiest fix was simply to make the tests completely serial (by limiting the workers to 1 in all cases).